src/Notification/NotificationFactory.php

```diff
@@ -6,6 +6,7 @@
 use Fandom\Includes\Util\UrlUtilityService;
 use Fandom\WikiConfig\WikiVariablesDataService;
 use MediaWiki\User\UserFactory;
+use Reverb\Identifier\IdentifierService;
 use WikiDomain\WikiConfigDataService;
 
 class NotificationFactory {
```

Comment:
Is there a test for this class?

Reply:
No, in this extension there is only one test class (for `IdentifierService`)